modules/battery_manager/test/BatteryManagerTest.php

```diff
@@ -324,7 +324,7 @@ function testFilter()
             self::$display,
             self::$clearFilter,
             '0',
-            '2 rows'
```

Comment:
This test was failing and when I investigated I don't see any rows with a value of "0" in RB and nothing inserted from the test, so I'm not sure where "2" was coming from and I think it's an error.

Reply:
That's a good idea. I'll try.